src/controllers/execute.ts

```diff
@@ -26,15 +31,19 @@ export const monitorQuote = async (ctx: Context): Promise<Context> => {
 
 export const monitorTransfer = async (ctx: Context): Promise<Context> => {
   try {
+    const span = apm.startSpan('transfer');
+
     const reqData = ctx.request.body as Record<string, unknown>;
     const param: FlowFileRequest = new FlowFileRequest();
-    reqData["TransactionType"] = "pacs.008.001.10";
-    let objJsonB64 = Buffer.from(JSON.stringify(reqData)).toString("base64");
+    reqData.TransactionType = 'pacs.008.001.10';
+    const objJsonB64 = Buffer.from(JSON.stringify(reqData)).toString('base64');
     param.setContent(objJsonB64);
     const resp = await nifiService.send(param);
 
     ctx.status = 200;
     ctx.body = { result: resp.getBody() };
+
+    span?.end();
```

Comment:
Do you think we should still end the span's even if there is an exception? You have to instantiate the span outside the "try-catch" then we can still span?.end()  in the Catch - this way we still have some APM even if exception happens? What do you think?

Reply:
There will be an improvement (fixes) on the next brach.